addons/project/data/project_demo.xml

```diff
@@ -1313,6 +1314,79 @@ Send it ASAP, its urgent.</field>
             <value eval="{'stage_id': ref('project_personal_stage_demo_6')}"/>
         </function>
 
+        <!-- Mail Message Notification -->
```

Comment:
Could you use a slightly more descriptive comment? Something like
```suggestion
        <!-- Change task creation notifications date -->
```

Reply:
Hello @MissingNoShiny 
Thank you for looking into it. :slightly_smiling_face: 
I have implemented the changes